src/Orchestrator/appsettings.json

```diff
@@ -40,6 +40,10 @@
     "ApiKey": "",
     "Address": ""
   },
+  "GatewayConfig": {
+    "NodePort": 31000,

```

Comment:
Please leave these values empty, so they will not be used for everyone.

Reply:
changed isIngress to false, left comment with range of NodePort